pettingzoo/utils/wrappers/base.py

```diff
@@ -34,19 +40,17 @@ def __init__(self, env):
             pass
 
         # Not every environment has the .state_space attribute implemented
-        try:
-            self.state_space = self.env.state_space
-        except AttributeError:
-            pass
+        if hasattr(self.env, "state_space"):
+            self.state_space = self.env.state_space  # type: ignore
```

Comment:
Why do we need a `type: ignore` here? It might be better to stick with the previous version if that's necessary

Reply:
Reverted. I tried to change it with hasattr to see if the pyright error went away and then forgot to revert it.